### ISSUE TITLE
`processSend` should not be promise

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2648]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.34.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.34.3...4.34.4) (2022-04-11)

- Release 4.34.4 [`ad8fe84`](https://github.com/open-wa/wa-automate-nodejs/commit/ad8fe8436abc3f1c13e2be1da02a27a523e30330)
- 🚑 fix: #2648 [`482183a`](https://github.com/open-wa/wa-automate-nodejs/commit/482183a22eba1907a50432cf684b81738d4df3ec)
- updated types-only package [`7abab32`](https://github.com/open-wa/wa-automate-nodejs/commit/7abab32da9ed6e845af40d89adeda04242422094)
